packages/eas-cli/src/commands/build/index.ts

```diff
@@ -79,6 +81,11 @@ export default class Build extends EasCommand {
       helpValue: 'PROFILE_NAME',
       exclusive: ['auto-submit'],
     }),
+    'resource-class': Flags.enum({
+      options: Object.values(UserInputResourceClass),
+      hidden: true,
+      description: 'The resource class that will be used to run this build [experimental]',
```

Comment:
resources class term is common in CI ecosystem, but for users it might not be obvious what it means. It's fine as a name of the option, but the description should use sth else to define it e.g `The instance type that will be used to run this build`